src/spyglass/histology/v1/histology.py

```diff
@@ -0,0 +1,141 @@
+    pixel_size_y: float                 # (um) Pixel size in Y dimension after processing/scaling
+    pixel_size_z: float                 # (um) Pixel size in Z dimension (often slice_thickness or scan step)
+    objective_magnification: float      # Magnification of the objective lens (e.g., 20 for 20x)
+    image_modality: enum(               # Modality of the microscopy
+        "fluorescence",
+        "brightfield",
+        "other"
+    )
+    processing_notes="": varchar(1024)  # Notes on image processing applied before/during NWB creation
+    """
+
+    # Ensure this key is unique for HistologyImages entries
+    # key_source = Histology
+
+    def make(self, key: dict) -> None:
+        """
+        Populate HistologyImages table.
+        This method should:
+        1. Find the raw image data using `raw_scan_path` from the `Histology` table.
+        2. Process the images as needed (e.g., stitching, scaling).
+        3. Create an NWB file containing the processed image stack (e.g., using `pynwb.image.ImageSeries`).
+           - Store relevant metadata (pixel sizes, objective, modality, etc.) within the NWB file.
+        4. Create an `AnalysisNwbfile` entry for the new NWB file.
+        5. Insert the key, including the `analysis_file_name` from AnalysisNwbfile,
+           along with image metadata like `pixel_size_*`, `color_to_stain`, etc., into this table.
+        """
+        logger.info(f"Populating HistologyImages for key: {key}")
+        # Placeholder: Replace with actual NWB creation and insertion logic
+        # Example steps (conceptual):
+        # 1. histology_entry = (Histology & key).fetch1()
+        # 2. raw_path = histology_entry['raw_scan_path']
+        # 3. image_data, metadata = process_histology_images(raw_path) # Your function
+        # 4. nwb_file_name = f"{key['subject_id']}_{key['histology_id']}_images.nwb"
+        # 5. nwb_file_path = AnalysisNwbfile().create(nwb_file_name)
+        # 6. create_histology_nwb(nwb_file_path, image_data, metadata) # Your function
+        # 7. AnalysisNwbfile().add(key['subject_id'], nwb_file_name)
+        # 8. self.insert1({
+        #     **key,
+        #     'images_id': key['histology_id'], # Or generate a new unique ID if needed
+        #     'analysis_file_name': nwb_file_name,
+        #     'pixel_size_x': metadata['pixel_size_x'],
+        #     # ... other metadata fields ...
+        # })
+        pass
+
+
+@schema
+class HistologyRegistration(SpyglassMixin, dj.Manual):
+    definition = """
+    # Stores results/params of aligning histology image data to a target coordinate system
+    -> HistologyImages                  # Link to the source histology NWB file info
+    registration_id: varchar(32)        # Unique ID for this registration instance/parameters
+    ---
+    -> BrainCoordinateSystem                 # The TARGET coordinate system (e.g., 'allen_ccf_v3_ras_um')
+
+    # --- Registration Parameters ---
```

Comment:
I'd consider either ...
1. making these part tables with only `-> master` as the primary key. At the cost of violating the one-to-many assumption of master-parts, you get fewer fields and the ability to shorten names (`HistologyRegistration.registration_X` -> `HistologyRegistration.Params.X`)
2. making the params a lookup table